src/config/default.js

```diff
@@ -0,0 +1,44 @@
+import collector from './../collector';
+
+const { getHostname, getCollectorPath } = collector;
+
+export default class DefaultConfig {
+  /**
+   * Default configuration
+   *
+   * This class should define sensible defaults for any supported config
+   * values.
+   */
+
+  get clientId() {
+    return '';
```

Comment:
@kolanos why the empty string and not `undefined` ?

Reply:
@coreylight That was the previous behavior.